x/gov/module.go

```diff
@@ -112,6 +112,18 @@ func (am AppModule) ExportGenesis(ctx sdk.Ctx) json.RawMessage {
 
 // module begin-block
 func (am AppModule) BeginBlock(ctx sdk.Ctx, req abci.RequestBeginBlock) {
+	
+	if am.keeper.GetCodec().IsOnNamedFeatureActivationHeight(ctx.BlockHeight(), codec.RSCALKey) {
+		gParams := am.keeper.GetParams(ctx)
```

Comment:
This variable should be named `params` to mantain consistent nomenclature with other places where `GetParams` is called.

Reply:
address in https://github.com/pokt-network/pocket-core/pull/1449/commits/8f4f0acaac4df17338bd685c442744fb2488a2c1